### ISSUE TITLE
Highlight reported player in-game and chat

### ISSUE THREAD

**Issue by Eliofx92** [runelite/runelite#11274]
Can you please highlight players ingame and chat that we have reported?

for example if I report someone for spamming a website can their text + name be red in the chat box?

I know we can add the user to the ignore list but the max limit is quite small so wont be any use.

Cheers

**Comment by Nightfirecat** [comment]
I think this would be better solved by adding users to the "Filtered Names" box of the Chat Filter plugin.

**Comment by Eliofx92** [comment]
Can't you make the report feature automatically add users to the "Filtered Names" box of the Chat Filter plugin please? There's far too many spammers to do it manually.

Thanks for your time.

**Comment by Nightfirecat** [comment]
If you're just looking to block spammers, I'd recommend instead adding the phrases they spam into the other fields of the Chat Filter plugin. With a good set of filters, you'll never see their messages again.

**Comment by Eliofx92** [comment]
> If you're just looking to block spammers, I'd recommend instead adding the phrases they spam into the other fields of the Chat Filter plugin. With a good set of filters, you'll never see their messages again.

I am actually a player moderator so I don't want to block spammers automatically as its my job to report them and mute them. It would also be beneficial for me and regular players to see who we have reported in the past so we can be aware of them.

Removing messages of past reported players would be useful when I am playing my alt when I haven't got the ability to mute them via my moderator tools. Most Runescape players would appreciate these suggestions especially if they are optional on runelite :)

**Comment by nlruk** [comment]
Good suggestion. Massive problem for f2p players

**Comment by Tengauge12** [comment]
Stealing peoples ideas (Mine) and turning it into something else. Lol.

The original idea (By me; a couple years ago)

When someone is reported, their name turn blue (Those on your ignore list) opposed to those who show up green (Those on your friends list)

**Comment by Tengauge12** [comment]
https://www.reddit.com/r/2007scape/comments/aqvw84/helpful_bot_report_interface/

**Comment by melkypie** [comment]
I agree with nightfire about the chat filters, if you are looking to just mark out spammers messages or remove them entirely.
If there was something like this to be added, the list of names that you have reported can easily get out of date (if they are not on your ignore/friends list) due to people having the ability to name change and there is no real way of tracking that besides having that person on the ignore/friends list.
Also I am pretty sure the message of a person who is on your ignore list is hidden from chat so there would be no way to mark someone in chat if they didn't send any messages. If you want it to mark out people in your clan chat that are on your ignore list, there is already an option for that in clan chat plugin

**Comment by Eliofx92** [comment]
> Stealing peoples ideas (Mine) and turning it into something else. Lol.
> 
> The original idea (By me; a couple years ago)
> 
> When someone is reported, their name turn blue (Those on your ignore list) opposed to those who show up green (Those on your friends list)

I don't know if you're joking but I did not steal your idea. Obviously there is a demand if other people are wanting it too.
